packages/admin/cms-admin/src/dam/DataGrid/duplicatedFilenames/ManuallyHandleDuplicatedFilenamesDialog.tsx

```diff
@@ -79,6 +87,26 @@ export const ManuallyHandleDuplicatedFilenamesDialog = ({ open, filenameData, on
                         }}
                     />
                 </Button>
+                <Button variant="contained" color="warning" onClick={onReplace}>
+                    <FormattedMessage
+                        id="comet.dam.duplicateFilenameDialog.action.replaceFiles"
+                        defaultMessage="Replace {count, plural, one {file} other {files}}"
+                        values={{
+                            count: filenameData.length,
+                        }}
+                    />
+                    <Tooltip
```

Comment:
The tooltip should probably be for the button, not the icon

Reply:
no more tooltip 493954e10b7cc08cd886f0ffa6a9699c7b16c555 now with the new design using an alert dialog